src/main/java/com/qaprosoft/carina/demo/gui/pages/AutomationPracticeContactControllerPage.java

```diff
@@ -0,0 +1,21 @@
+package com.qaprosoft.carina.demo.gui.pages;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class AutomationPracticeContactControllerPage extends AbstractPage {
```

Comment:
Why we need this page?

Reply:
It is the step after sending the contact form. It redirects you to an intermediate step where it says that the message has been sent and there is a button that redirects to home page

Reply:
I think we don't need to create separate page object for this. This "Your message has been successfully sent to our team." element can be added to `AutomationPracticeContactPage.java`